src/Depressurizer.Core/Helpers/Locations.cs

```diff
@@ -19,6 +19,11 @@ public static class File
             /// </summary>
             public static string Database => "database.json";
 
+            /// <summary>
+            ///     Default database (gzipped) file location.
+            /// </summary>
+            public static string DatabaseGzip => "database.json.gz";
```

Comment:
I would suggest doing something similar to this:

```suggestion
            public static string DatabaseGzip => Database + ".gz";
```